Gemfile

```diff
@@ -37,6 +37,8 @@ gem "rbs", require: false
 gem "anyway_config"
 gem "slim"
 gem "redis-session-store"
+gem "dry-struct"
+gem "factory_bot"
```

Comment:
Any reason you're including FactoryBot in the production environment? I've always seen this in the "development & test" group.

Reply:
No reason, I will update the Gemfile 👍🏻